### PR TITLE
fix: polish typos fix and index pointing to beta.2

### PR DESCRIPTION
**Pull request by Morishiri**
Some fixes as I missed some things last time.

### CODE REVIEW

**Comment by damianopetrungaro** [comment]
@Morishiri  i see a lot of changes, i'll run it locally with the docker-compose setup.
Could you please double check it too?

Thank you!

**Comment by damianopetrungaro** [comment]
Hey @Morishiri  just checked.
Your PR broke the pl pages, let me add some reviews so you can fix it 😄

File: lang/pl/index.md [hunk 1]
```diff
@@ -1,42 +1,67 @@
...
 language: pl
 ---
 
-# Konwencjonalne Wiadomości 1.0.0-beta.1
+# Konwencjonalne Commity 1.0.0-beta.2
 
 ## Streszczenie
 
-Jako opiekun otwartych źródeł, spłaszczaj gałęzie funkcyjne (featurowe) do gałęzi głównej - `master`
+Jako opiekun otwartych źródeł, spłaszczaj (squashuj) gałęzie funkcyjne (featurowe) do gałęzi głównej - 'master'
```
Comment (damianopetrungaro) [inline lang/pl/index.md]: Rollback to `master`

File: lang/pl/index.md [hunk 2]
```diff
@@ -1,42 +1,67 @@
...
 oraz pisz ustandaryzowane opisy dostarczeń (commitów).
 
 Opis dostarczenia powinien wyglądać następująco:
 
 ---
 
-```
-<typ>[opcjonalny zakres]: <opis>
+'''
```
Comment (damianopetrungaro) [inline lang/pl/index.md]: Use ` ``` ` and not `'''`

File: lang/pl/index.md [hunk 3]
```diff
@@ -1,42 +1,67 @@
...
+<typ&gt;[opcjonalny zakres]: <opis&gt;
```
Comment (damianopetrungaro) [inline lang/pl/index.md]: Change `&gt;` with `>`

File: lang/pl/index.md [hunk 4]
```diff
@@ -1,42 +1,67 @@
...
 
 [opcjonalne ciało wiadomości]
 
 [opcjonalna stopka wiadomości]
-```
+'''
```
Comment (damianopetrungaro) [inline lang/pl/index.md]: Use ``` and not '''

File: lang/pl/index.md [hunk 5]
```diff
@@ -1,42 +1,67 @@
...
 ---
 
-<br />
+
```
Comment (damianopetrungaro) [inline lang/pl/index.md]: Add `<br />`

File: lang/pl/index.md [hunk 6]
```diff
@@ -1,42 +1,67 @@
...
 Wiadomość zawiera następujące strukturalne elementy po to, aby zakomunikować intencje
 do użytkowników Twojej biblioteki:
 
-1. **fix:** dostarczenie _typu_ `fix` naprawia błąd obecny w Twoim kodzie (powiązane z [`PATCH`](http://semver.org/#summary) w wersjonowaniu semantycznym).
-2. **feat:** dostarczenie _typu_ `feat` wprowadza nowe funkcje do Twojej biblioteki (powiązane z [`MINOR`](http://semver.org/#summary) w wersjonowaniu semantycznym).
-3. **BREAKING CHANGE:** dostarczenie, które posiada tekst `BREAKING CHANGE:` na początku jego opcjonalnego ciała bądź stopki wprowadza zmianę łamiącą kompatybilność API (powiązane z [`MAJOR`](http://semver.org/#summary) w wersjonowaniu semantycznym). Zmiana łamiąca kompatybilność wsteczną może być elementem zmian każdego innego _typu_, np. `fix:`, `feat:` & `chore:` - wszystkie byłyby poprawne, w dodatku do każdego innego _typu_.
+1. **fix:** dostarczenie _typu_ 'fix' naprawia błąd obecny w Twoim kodzie (powiązane z ['PATCH'](http://semver.org/#summary) w wersjonowaniu semantycznym).
```
Comment (damianopetrungaro) [inline lang/pl/index.md]: Update all the `'` with  `

**Comment by Morishiri** [comment]
hmm, looks like my editor did something strange while I copied this to spellchecker...

**Comment by damianopetrungaro** [comment]
No worries :D 
Tag me when you'll complete it 😄

**Comment by Morishiri** [comment]
I think it looks fine now.

File: lang/pl/index.md [hunk 7]
```diff
@@ -30,13 +30,38 @@ do użytkowników Twojej biblioteki:
...
 3. **BREAKING CHANGE:** dostarczenie, które posiada tekst `BREAKING CHANGE:` na początku jego opcjonalnego ciała bądź stopki wprowadza zmianę łamiącą kompatybilność API (powiązane z [`MAJOR`](http://semver.org/#summary) w wersjonowaniu semantycznym). Zmiana łamiąca kompatybilność wsteczną może być elementem zmian każdego innego _typu_, np. `fix:`, `feat:` & `chore:` - wszystkie byłyby poprawne, w dodatku do każdego innego _typu_.
+4. Others: commity _typu_ innego niż `fix:` oraz `feat:` są dozwolone, np. [commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional) (bazowano na [konwencji Angulara](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)) poleca `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, oraz inne. My polecamy także `improvement` dla dostarczeń, które ulepszają obecną implementację bez dodawania nowych funkcjonalności lub naprawy błędów. Zauważ, że te typy nie są obowiązkowe według specyfikacji konwencjonalnych commitów i nie mają wpływu na wersjonowanie (chyba, że zawierają BREAKING CHANGE, co NIE JEST rekomendowane).
 
-<br />
```
Comment (damianopetrungaro) [inline lang/pl/index.md]: Do not remove `<br />` here 😄

File: lang/pl/spec/v1.0.0-beta.1.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ do użytkowników Twojej biblioteki:
 2. **feat:** dostarczenie _typu_ `feat` wprowadza nowe funkcje do Twojej biblioteki (powiązane z [`MINOR`](http://semver.org/#summary) w wersjonowaniu semantycznym).
 3. **BREAKING CHANGE:** dostarczenie, które posiada tekst `BREAKING CHANGE:` na początku jego opcjonalnego ciała bądź stopki wprowadza zmianę łamiącą kompatybilność API (powiązane z [`MAJOR`](http://semver.org/#summary) w wersjonowaniu semantycznym). Zmiana łamiąca kompatybilność wsteczną może być elementem zmian każdego innego _typu_, np. `fix:`, `feat:` & `chore:` - wszystkie byłyby poprawne, w dodatku do każdego innego _typu_.
 
-<br />
+
```
Comment (damianopetrungaro) [inline lang/pl/spec/v1.0.0-beta.1.md]: Do not remove `<br />` here 😄

File: lang/pl/spec/v1.0.0-beta.2.md [hunk 9]
```diff
@@ -27,12 +27,11 @@ Opis dostarczenia powinien wyglądać następująco:
...
-<br />
+4. Others: commity _typu_ innego niż `fix:` oraz `feat:` są dozwolone, np. [commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional) (bazowano na [konwencji Angulara](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)) poleca `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, oraz inne. My polecamy także `improvement` dla dostarczeń, które ulepszają obecną implementację bez dodawania nowych funkcjonalności lub naprawy błędów. Zauważ, że te typy nie są obowiązkowe według specyfikacji konwencjonalnych commitów i nie mają wpływu na wersjonowanie (chyba, że zawierają BREAKING CHANGE, co NIE JEST rekomendowane).
+
```
Comment (damianopetrungaro) [inline lang/pl/spec/v1.0.0-beta.2.md]: Do not remove `<br />` here 😄

**Comment by damianopetrungaro** [comment]
@Morishiri here we go!
Thank you for another contribution!

**Pull request merged**